pontoon/base/templates/locale_contributors.html

```diff
@@ -0,0 +1,18 @@
+{% extends 'locale.html' %}
+{% import 'widgets/contributors.html' as Contributors %}
+
+{% block title %}{{ super() }} - contributors {% endblock %}
```

Comment:
Instead of dash and lowercase (- contributors), let's keep the pattern and use: &middot; Contributors.